pt_br/install.md

```diff
@@ -0,0 +1,126 @@
+
+
+## Configure seu Editor
+
+Usar Elm é muito mais legal quando você tem um editor de código para te ajudar. Existem plugins de Elm disponíveis para pelo menos os seguintes editores:
+
+  * [Atom](https://atom.io/packages/language-elm)
+  * [Brackets](https://github.com/lepinay/elm-brackets)
+  * [Emacs](https://github.com/jcollard/elm-mode)
+  * [IntelliJ](https://github.com/durkiewicz/elm-plugin)
+  * [Light Table](https://github.com/rundis/elm-light)
+  * [Sublime Text](https://packagecontrol.io/packages/Elm%20Language%20Support)
+  * [Vim](https://github.com/lambdatoast/elm.vim)
+  * [VS Code](https://github.com/sbrink/vscode-elm)
+
+Se você não tem nenhum editor ainda, o [Sublime Text](https://www.sublimetext.com/) é uma boa pedida para começar!
+
+Você também deveria tentar utilizar o [elm-format][] que deixa o seu código bem mais bonito!
+
+[elm-format]: https://github.com/avh4/elm-format
+
+
+## As ferramentas em linha de comando
+
+Então, instalamos o Elm e ele nos deu `elm-repl`, `elm-reactor`, `elm-make`, e `elm-package`. Mas o que exatamente tudo isso faz?
+
+### elm-repl
+
+[`elm-repl`](https://github.com/elm-lang/elm-repl) permite que você brinque com as expressões em Elm
+
+```bash
+$ elm-repl
+---- elm-repl 0.18.0 -----------------------------------------------------------
+ :help for help, :exit to exit, more at <https://github.com/elm-lang/elm-repl>
+--------------------------------------------------------------------------------
+> 1 / 2
+0.5 : Float
+> List.length [1,2,3,4]
+4 : Int
+> String.reverse "stressed"
+"desserts" : String
+> :exit
+$
+```
+
+Nós usaremos o `elm-repl` na próxima seção &ldquo;Core da Linguagem&rdquo;. Você pode ler mais sobre como ele funciona [aqui](https://github.com/elm-lang/elm-repl/blob/master/README.md).
+
+> **Nota:** `elm-repl` funciona compilando código para JavaScript, então certifique-se de ter o [Node.js](http://nodejs.org/) instalado, pois ele será utilizado para avaliar o código.
+
+
+### elm-reactor
+
+[`elm-reactor`](https://github.com/elm-lang/elm-reactor) te ajuda a montar seus projetos em Elm sem precisar ficar mexendo demais com a linha de comando. Você simplesmente executa ele na raiz de seu projeto, assim:
+
+```bash
+git clone https://github.com/evancz/elm-architecture-tutorial.git
+cd elm-architecture-tutorial
+elm-reactor
+```
+Ele vai iniciar um servidor em [`http://localhost:8000`](http://localhost:8000). Você pode navegar por quaisquer arquivos Elm e ver como eles são. Dê uma olhada em `examples/01-button.elm`.
+
+**Flags importantes:**
+
+- `--port` te deixa escolher uma porta diferente da porta 8000. Ou seja, você pode escrever
+  `elm-reactor --port=8123` para fazer as coisas rodarem em `http://localhost:8123`.
+- `--address` te deixa substituir `localhost` por algum outro endereço. Por exemplo, você talvez queira utilizar `elm-reactor --address=0.0.0.0` se quiser testar um programa em Elm em um disposivo móvel conectado à sua rede local.
+
+## elm-make

```

Comment:
All the other sections on `As ferramentas em linha de comando`  has `<h3>` headers only that is `<h2>`.

I know it's like that on the original. I'll open a PR to the original repo.